docs/developer-docs/latest/developer-resources/database-migrations.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Database migrations - Strapi Developer Docs
+description: Strapi database migrations are ways to modify the database
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/development/backend-customization/database-migrations.html
+---
+
+# Database migrations (experimental feature)
+
+Database migrations exist to run one-time queries against the database, typically to modify the tables structure or the data when upgrading the Strapi app.
+
+::: callout 🚧  Experimental
+The feature is to be considered as unfinished and experimental.
+:::
+
+## General logic
+
+Migration are run using javascript migration files stored in `database/migrations`.
+
+Migration files should export 2 functions `up()` and `down()`.
```

Comment:
```suggestion
Migration files should export 2 functions `up()` and `down()`:
```

Reply:
Not sure what these symbols are but probably best to not include them
![image](https://user-images.githubusercontent.com/8593673/167686872-2080c429-5a3c-4510-9dd6-d1b82393ddc7.png)

Reply:
Yeah, not sure what happened, had to remove some in other places yesterday and my commit was supposed to remove them here, not _add_ them… 😂   🤷 
Thanks for the heads up, Derrick!